src/v3/src/components/Title/Title.tsx

```diff
@@ -12,27 +12,46 @@
 
 import { Box, Typography } from '@okta/odyssey-react-mui';
 import { h } from 'preact';
+import { useEffect, useRef } from 'preact/hooks';
 
+import { useWidgetContext } from '../../contexts';
 import { TitleElement, UISchemaElementComponent } from '../../types';
+import style from './style.css';
 
 const Title: UISchemaElementComponent<{
   uischema: TitleElement
 }> = (
   { uischema: { id, options } },
-) => (
-  <Box
-    display="flex"
-    justifyContent="flex-start"
-  >
-    <Typography
-      id={id}
-      component="h2"
-      variant="h4"
-      data-se="o-form-head"
+) => {
+  const titleRef = useRef<HTMLTitleElement>(null);
+  const { widgetProps } = useWidgetContext();
+  const { features: { autoFocus = false } = {} } = widgetProps;
+
+  useEffect(() => {
+    if (!autoFocus) {
+      titleRef.current?.focus();
+    }
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  return (
+    <Box
+      display="flex"
+      justifyContent="flex-start"
     >
-      {options?.content}
-    </Typography>
-  </Box>
-);
+      <Typography
+        id={id}
+        className={style.noOutline}
```

Comment:
Can you add this type in `sx` rather than using `className`?

Reply:
ah yes we can good call.  i forgot about that `sx` prop lol